tests/registry_interface.rs

```diff
@@ -474,6 +520,10 @@ mod interface_tests {
         tl4.insert("tag".to_string());
         tl4.insert("three".to_string());
         println!("Running check_tag_list_n_last 4");
-        check_tag_list_n_last(&client, 2, "latest", &tl4)
+        check_tag_list_n_last(&client, 2, "latest", &tl4);
+        
+        get_readiness(&client);
+        get_health(&client);
+        get_metrics(&client);
```

Comment:
Could we add a test at the start that gets the diskspace metric, then at the end run it again and test that the value has changed (diskspace has been consumed)?

Also, add `println!` statements to help debugging. These tests should really be in separate test methods, but that causes problems with need to stop and restart and the registry, hence the `test_runner`. One of the downsides is that there's less information on what test is running, so the printlns help.

Reply:
Added the print line. Not sure about the disk differences test because disk test though.